queuebacca-sqs/src/main/java/io/axonif/queuebacca/sqs/SqsCourierMessageBinRegistry.java

```diff
@@ -232,6 +234,10 @@ private void setTags(String queueUrl, Collection<SqsTag> tags, Configuration con
                     throw e;
                 }
             }
+        } while (++attempts < configuration.getInt(SQS_TAGGING_RETRY_ATTEMPTS_PROPERTY, SQS_TAGGING_RETRY_ATTEMPTS_DEFAULT));
+
+        if (attempts == SQS_TAGGING_RETRY_ATTEMPTS_DEFAULT) {
```

Comment:
This shouldn't compare against the default, it should compare against the property value.

Reply:
Yea I already caught it, I just hadn't pushed yet.